--- a/routes/b2bOrder.ts
+++ b/routes/b2bOrder.ts
@@ -20,7 +20,10 @@
       try {
         const sandbox = { safeEval, orderLinesData }
         vm.createContext(sandbox)
-        vm.runInContext('safeEval(orderLinesData)', sandbox, { timeout: 2000 })
+         const result = sandbox.safeEval(orderLinesData);
+         if (result !== undefined) {
+           res.json({ cid: body.cid, orderNo: uniqueOrderNumber(), paymentDue: dateTwoWeeksFromNow() })
+         }
         res.json({ cid: body.cid, orderNo: uniqueOrderNumber(), paymentDue: dateTwoWeeksFromNow() })
       } catch (err) {
         if (utils.getErrorMessage(err).match(/Script execution timed out.*/) != null) {
